valohai/internals/utils.py

```diff
@@ -6,3 +6,15 @@ def string_to_bool(value: str) -> bool:
     if value.lower() == "false":
         return False
     return True
+
+
+def check_sha256_hashing(filepath: str) -> str:
+    try:
+        from valohai_cli.utils.hashing import get_fp_sha256  # type: ignore
+    except ImportError as ie:
+        raise RuntimeError(
+            "The `valohai-cli` module must be available " "for verifying hash"
+        ) from ie
+
+    with open(filepath, "rb") as f:
+        return str(get_fp_sha256(f))
```

Comment:
`get_fp_sha256()` already always returns a `str`:
```suggestion
        return get_fp_sha256(f)
```